R/convertToEGRET.R

```diff
@@ -0,0 +1,75 @@
+#' Convert loadflex to EGRET object
+#'  
+#' Convert a loadflex object into an EGRET object for plotting.
+#' 
+#' @param intdat data.frame of interpolation data
+#' @param estdat data.frame of estimation data
+#' @param preds data.frame of load predictions
+#' @param meta loadflex metadata object; it must include constituent,
+#' flow, conc.units, custom (station abbreviation: staAbbr)
+#' 
+#' @importFrom dplyr left_join
+#' @importFrom dplyr rename_
+#' @importFrom dplyr mutate
+#' @importFrom lubridate decimal_date
+#' @importFrom EGRET as.egret
+#' 
+#' @export
```

Comment:
does this need to be exported?

if we did export it, would this function already be everything you'd need to run an EGRET model on the same data?